projects/kit/components/select/select.style.less

```diff
@@ -16,18 +16,23 @@
     /* stylelint-enable */
 }
 
-.hosted {
+.t-hosted {
     display: block;
     border-radius: inherit;
 }
 
-.textfield {
+.t-textfield {
     border-radius: inherit;
     text-align: inherit;
 }
 
-.value {
+.t-value {
     display: flex;
     width: 100%;
     align-items: center;
+    overflow: hidden;
+
+    &__primitive {
```

Comment:
nit: we use single `_` symbol and we use it designate some status. Like `t-value_focused` or `t-value_error`, but not that it is nested in `t-value`. With built-in encapsulation in Angular we try to make our style sheets flat.

Reply:
@waterplea that is, should I rename to `t-value-primitive`?

Reply:
Probably `t-primitive-value` is more correct

Reply:
fixed